### PR TITLE
Enable some of the python CacheCompat tests to run

### PR DESCRIPTION
**Pull request by MarkZuber**
I've enabled "pip install msal" on the PR/CI Nightly builds so that we can regularly run cache compat tests with MSAL.NET and MSAL.Python.  I validated that this works in a separate test build pipeline and I've now added those changes to our PR/CI pipelines (Thanks @trwalke!) 

Only the "Msal Python -> MSAL/ADAL" tests are working since Msal.Python doesn't yet support authority aliasing so it can't find the network-cache-keyed values in there yet, but they're working on that.  Once those are available, we can add those tests back in.

### CODE REVIEW

**Comment by MarkZuber** [comment]
Integration tests don't yet run on the hosted agent.  So I've disabled the python part and will hold this PR until we get the last pieces in place for integration tests to run hosted.

File: tests/CacheCompat/CommonCache.Test.MsalPython/CommonCache.Test.MsalPython.csproj [hunk 1]
```diff
@@ -1,3 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="Current">
   <Import Project="../CacheTest.props" />
+  <ItemGroup>
+    <None Update="TestMsalPython.py">
```
Comment (bgavrilMS) [inline tests/CacheCompat/CommonCache.Test.MsalPython/CommonCache.Test.MsalPython.csproj]: Is this file already checked in?

Comment (MarkZuber) [reply tests/CacheCompat/CommonCache.Test.MsalPython/CommonCache.Test.MsalPython.csproj]: yes, it just wasn't in the project definition and it needs to be copied to the output on builds.

**Comment by MarkZuber** [comment]
Follow up build worked  https://identitydivision.visualstudio.com/IDDP/_build/results?buildId=315247

**Pull request merged**